source/modulo_new_core/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+cmake_minimum_required(VERSION 3.15)
+project(modulo_new_core)
+
+# default to C99
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 99)
+endif()
+
+# default to C++17
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 17)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# find dependencies
+find_package(ament_cmake_auto REQUIRED)
+find_package(ament_cmake_python REQUIRED)
+
```

Comment:
Shouldn't there be a `find_package(control_libraries ...)` and a `find_library(clproto)` somewhere here, or am I missing something?